src/Request/BuildingProgressRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Justimmo\Api\Request;
+
+use Justimmo\Api\Entity\Realty\BuildingProgress;
+
+/**
+ *
+ * @method $this withCompletionDate()
+ *
+ * @method $this sortByName($direction = BaseApiRequest::ASC)
+ * @method $this sortByCompletionDate($direction = BaseApiRequest::ASC)
+ */
+class BuildingProgressRequest extends BaseApiRequest
```

Comment:
There is no api call to fetch the building progress. This request would always return a 404.